### PR TITLE
Feature/veritable docker

### PR DESCRIPTION
**Pull request by mattdean-digicatapult**
Adds veritable to the 3 persona docker compose. To do this I've split up the docker-compose-3-persona insto several files to make reading it easier. I've also added additoinal ipfs nodes as we're now going to be using ipfs more seriously. Note this has increased resource usage in docker significantly and we might want to look at that before merging. Locally without doing much I'm now using nearly 7GB of RAM 😬

### CODE REVIEW

**Review by n3op2 — APPROVED** [review]
lgtm, have not tested locally tho

**Comment by rmlearney-digicatapult** [comment]
LGTM. Scripts running locally except known issue with number 5 - revocation

File: scripts/setup_veritable.sh [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+  )
+  if [ $? -ne 0 ] ; then
+    echo "Error issuing credential"
+    exit 1
+  fi
+  local credential_state=$(echo $RESPONSE | jq -r .state )
+
+  while [ "$credential_state" != "done" ]
+  do
+    echo "Waiting for credential issuence to be done: $credential_state"
```
Comment (rmlearney-digicatapult) [inline scripts/setup_veritable.sh:175]: issuance

File: scripts/setup_veritable.sh [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+      curl -s -X 'GET' "http://localhost:$port_holder/credentials" -H 'accept: application/json'
+    )
+    if [ $? -ne 0 ] ; then
+      echo "Error getting credentials for $port_holder"
+      exit 1
+    fi
+    credential_state=$(echo $RESPONSE | jq -r  '.[0].state')
+    sleep 1
+  done
+  echo "Credential issued"
```
Comment (rmlearney-digicatapult) [inline scripts/setup_veritable.sh:186]: can we echo who we're issuing to?

**Pull request merged**